index.html

```diff
@@ -7,17 +7,34 @@
   </head>
   <body>
     <rise-playlist-item>
-      <rise-twitter id="demoComponent" screen-name="demo" hashtag="demo"></rise-twitter>
     </rise-playlist-item>
     <script src="<%= htmlWebpackPlugin.files.chunks['vendor'].entry %>"></script>
     <script src="<%= htmlWebpackPlugin.files.chunks['component'].entry %>"></script>
 
-    <script>
-      const risePlaylistItem = document.getElementsByTagName("rise-playlist-item")[0];
+    <div id="log" style="height: 450px; width: 500px; display:none;"></div>
+    <script src="http://rvashow.appspot.com/scripts/jquery-1.10.2.min.js" type="text/javascript"></script>
+    <script type="text/javascript" language="javascript" src="http://rvashow.appspot.com/scripts/viewerScripts.js"></script>
+    <script src="http://rvashow.appspot.com/scripts/primus.js" type="text/javascript"></script>
+    <script src="http://rvashow.appspot.com/scripts/primus-local-messaging.js" type="text/javascript"></script>
+    <script src="http://rvashow.appspot.com/dist/viewer.js"></script>
+    <script type="text/javascript">
+     document.addEventListener("DOMContentLoaded", function() {
+        window.disableViewerContentFetch = true;
+        window.RiseVision.Viewer.EntryPoint.init();
 
-      const event = new CustomEvent("play");
+        const risePlaylistItem = document.getElementsByTagName("rise-playlist-item")[0];
+        const RiseTwitter = customElements.get('rise-twitter');
+        const riseTwitter = new RiseTwitter();
+        riseTwitter.setAttribute("id", "demoComponent");
+        riseTwitter.setAttribute("screen-name", "demo");
```

Comment:
@rodrigopavezi We should change the `screen-name` and `hashtag` to RiseVision for now, thanks.